[CI] GenerativeMetricsIT test failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #11437 / ubuntu-2204_platform-support-unix](https://gradle-enterprise.elastic.co/s/nmouu54exefco)
- [elasticsearch-pull-request #104986 / part-3](https://gradle-enterprise.elastic.co/s/2zj6gmuospofw)
- [elasticsearch-periodic-platform-support #11421 / windows-2025_checkpart3_platform-support-windows](https://gradle-enterprise.elastic.co/s/uxx4uhkoy36qi)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.GenerativeMetricsIT.test" -Dtests.seed=BC522821090731B5 -Dtests.locale=en-KE -Dtests.timezone=Africa/Abidjan -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.GenerativeMetricsIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: query: ts k8*,k8s-do* | mv_expand event_city_boundary | dissect event_log "%{network.bytes_in}" | eval  eSWsbwZpawum = -1919092828, KwyVlRdYdn = network.cost + network.cost + events_received, event_city = @timestamp, IcdiOsfAOABG = to_lower(event), network.eth0.rx = -1544103042450030617, network.cost = -1280235984, iqjDoZHGyz = -1126008774196794053, `client.ip` = true | drop `network.bytes_in`, @timestamp, `events_received`, pod, `network.total_bytes_out`, eSWsbwZpawum, `network.eth0.tx`, `event_city_boundary`, network.total_cost, `@timestamp` | limit 864 | stats  jlIHjVRldk = avg(first_over_time(network.total_bytes_in)), network.eth0.firmware_version = sum(max_over_time(network.eth0.currently_connected_clients)) by xLaasNZEC = bucket(event_city,1hour)
exception: method [POST], host [http://[::1]:41947], URI [/_query?format=yaml&pretty=true&error_trace=true], status line [HTTP/1.1 400 Bad Request]
---
error:
  root_cause:
  - type: "verification_exception"
   
[truncated]
```

**Issue Reasons:**
- [main] 3 failures in test test (0.6% fail rate in 462 executions)
- [main] 2 failures in pipeline elasticsearch-periodic-platform-support (50.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -237,9 +237,6 @@ tests:
 - class: org.elasticsearch.search.basic.SearchWithRandomDisconnectsIT
   method: testSearchWithRandomDisconnects
   issue: https://github.com/elastic/elasticsearch/issues/138128
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeMetricsIT
-  method: test
-  issue: https://github.com/elastic/elasticsearch/issues/138311
 - class: org.elasticsearch.xpack.ml.integration.RegressionIT
   method: testTwoJobsWithSameRandomizeSeedUseSameTrainingSet
   issue: https://github.com/elastic/elasticsearch/issues/138319

--- a/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
+++ b/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
@@ -75,15 +75,21 @@ public abstract class GenerativeRestTest extends ESRestTestCase implements Query
         // Awaiting fixes for correctness
         "Expecting at most \\[.*\\] columns, got \\[.*\\]", // https://github.com/elastic/elasticsearch/issues/129561
 
-        // TS-command tests
+        // TS-command tests (acceptable errors)
         "time-series.*the first aggregation.*is not allowed",
         "count_star .* can't be used with TS command",
         "time_series aggregate.* can only be used with the TS command",
-        "implicit time-series aggregation function .* doesn't support type .*",
+        "implicit time-series aggregation function",
         "INLINE STATS .* can only be used after STATS when used with TS command",
         "cannot group by a metric field .* in a time-series aggregation",
-        "a @timestamp field of type date or date_nanos to be present",
-        "Output has changed from \\[.*\\] to \\[.*\\]" // https://github.com/elastic/elasticsearch/issues/134794
+        "@timestamp field of type date or date_nanos",
+        "which was either not present in the source index, or has been dropped or renamed",
+        "second argument of .* must be \\[date_nanos or datetime\\], found value \\[@timestamp\\] type \\[.*\\]",
+
+        // Ts-command errors awaiting fixes
+        "Output has changed from \\[.*\\] to \\[.*\\]", // https://github.com/elastic/elasticsearch/issues/134794
+        "Invalid call to dataType on an unresolved object \\?@timestamp", // https://github.com/elastic/elasticsearch/issues/140607
+        "expected named expression for grouping; got Bucket" // https://github.com/elastic/elasticsearch/issues/140606
     );
 
     public static final Set<Pattern> ALLOWED_ERROR_PATTERNS = ALLOWED_ERRORS.stream()

